article-ansible-devops-for-itops/article-ansible-devops-for-itops.md

```diff
@@ -0,0 +1,32 @@
+#**Ansible: DevOps for IT Ops**
+
+The following article is based on my experience as a middelware"IT Ops guys" and the benefits of Ansible.
+
+##**Background**
+Ansible is different to other automation tools like Chef, Puppet and Salt. The main difference is that Ansible is first a push tool rather than a pull. In now way are the other tools mentioned inferior but Ansible offers an easier transision from no automation to fully automated.
```

Comment:
- "In no way" rather than "In now way"
- "easier transision" -> "easier transition"